core/components/footer/footer.tsx

```diff
@@ -70,29 +87,12 @@ export const Footer = async () => {
   ];
 
   return (
-    <ComponentsFooter
-      contactInformation={data.settings?.contact ?? undefined}
```

Comment:
We don't seem to be passing any contact information anymore. This feels like a regression